### PR TITLE
[FLOC-3808] add node, container and dataset counts to description of flocker cluster

### PR DESCRIPTION
**Pull request by avg-I**
Those are useful for reasoning about the cluster's benchmark results.

### CODE REVIEW

File: benchmark/_driver.py [hunk 1]
```diff
@@ -134,12 +136,27 @@ def driver(
         printing or storage.
     """
 
-    d = cluster.get_control_service(reactor).version()
-
-    def add_control_service(version, result):
+    control_service = cluster.get_control_service(reactor)
+    d = control_service(reactor).version()
```
Comment (zubron) [inline benchmark/_driver.py]: We don't need this line.

Comment (avg-I) [inline benchmark/_driver.py]: oh, oops!

**Comment by zubron** [comment]
So, just to clarify, this gives us the total number of containers and datasets across the entire cluster.
```
  "control_service": {
    "flocker_version": "1.9.0.dev1+1221.gde4c49f",
    "dataset_count": 30,
    "container_count": 30,
    "node_count": 2,
    "host": "54.69.92.161"
  }
```

It looks good, thanks @avg-I! Merge when the tests have passed :)

**Comment by avg-I** [comment]
Thank you for the review!

**Pull request merged**